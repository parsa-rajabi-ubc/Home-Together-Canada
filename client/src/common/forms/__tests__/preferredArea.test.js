/**
 * @Author:     Jeff Hatton
 * @Created:    2020.11.19
 *
 * @Description: Phone number input fields Component Snapshot test
 *
 */
import React from 'react';
import renderer from  'react-test-renderer'
import PreferredArea from "../preferredArea";

describe('PreferredArea.test.js', () => {
    describe('Snapshot test', () => {
        it("should render correctly regardless of properties", () => {

            //when
            const component = renderer.create(<PreferredArea/>).toJSON();

            //then
            expect(component).toMatchSnapshot();
        });
    })
})