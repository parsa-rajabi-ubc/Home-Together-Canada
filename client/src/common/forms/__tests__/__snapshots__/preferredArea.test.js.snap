// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreferredArea.test.js Snapshot test should render correctly regardless of properties 1`] = `
<div>
  <p>
    Areas I am interested in living
  </p>
  <div
    className="relative inline-block text-left"
  >
    <div
      className=""
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      <span
        className=""
      >
        <p
          aria-expanded="true"
          aria-haspopup="true"
          className=""
          id="options-menu"
        >
          Province
           ↓ 
        </p>
      </span>
    </div>
  </div>
  <div
    className="relative inline-block text-left"
  >
    <div
      className=""
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      <span
        className=""
      >
        <p
          aria-expanded="true"
          aria-haspopup="true"
          className=""
          id="options-menu"
        >
          city
           ↓ 
        </p>
      </span>
    </div>
  </div>
  <div
    className="relative inline-block text-left"
  >
    <div
      className=""
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      <span
        className=""
      >
        <p
          aria-expanded="true"
          aria-haspopup="true"
          className=""
          id="options-menu"
        >
          radius
           ↓ 
        </p>
      </span>
    </div>
  </div>
</div>
`;
